src/main/java/com/qapital/plaid/client/DefaultPlaidUserClient.java

```diff
@@ -50,6 +53,11 @@ public String getAccessToken() {
     }
 
     @Override
+    public Boolean isProductionAccount() {
```

Comment:
Yes. Smaller memory footprint. But most importantly no null pointer issues to worry about. If I saw this in code I could feel the need to check if the reply could ever be `null`.